bundle/src/main/java/com/adobe/acs/commons/workflow/process/impl/WorkflowDelegationStep.java

```diff
@@ -78,125 +103,217 @@
     private static final String TERMINATE_ON_DELEGATION = "terminateWorkflowOnDelegation";
 
     @Override
-    public void execute(WorkItem workItem, WorkflowSession workflowSession, MetaDataMap metadata)
-            throws WorkflowException {
-
-        Map<String, String> args = getProcessArgsMap(metadata);
-
-        String propertyName = args.get(WORKFLOW_PROPERTY_NAME);
-        String defaultWorkflowModelId = args.get(DEFAULT_WORKFLOW_MODEL);
-        boolean terminateOnDelegation = Boolean.parseBoolean(StringUtils.lowerCase(args.get(TERMINATE_ON_DELEGATION)));
-
+    public void execute(WorkItem workItem, WorkflowSession workflowSession, MetaDataMap metadata) throws WorkflowException {
+
+      // Get Configuration
+      Map<String, String> args = getProcessArgsMap(metadata);
+      String confPropertyName = args.get(CONF_WORKFLOW_PROPERTY_NAME);
```

Comment:
Any reason we add this configuration param? Arent the Granite Conf's expecting the property to be `cq:conf` and if we can move to Sling CAConf `sling:configRef`.. when wouldn't it be the default?

Is this so if you are using Conf's but dont want it to drive WF delegation, that you can leave it out of process args which will "force" it to look at content property?

Reply:
This appears fine actually - ive heard cq:conf is supported.